### ISSUE TITLE
Analytics New KPI dashboard request & KPI dashboard update request for [Team]

### ISSUE THREAD

**Issue by mtcA6** [department-of-veterans-affairs/va.gov-team#75886]
### What this form is for

Use this ticket template to request a new standardized KPI dashboard or an update to an existing KPI dashboard on Domo for your product. 

**Tell us what you need done:**
- [ ] [New Dashboards](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/.github/ISSUE_TEMPLATE/analytics-request-kpi-dashboard.md#for-new-dashboards)
- [x] [Updates to Existing Dashboards](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/.github/ISSUE_TEMPLATE/analytics-request-kpi-dashboard.md#for-updates-to-existing-dashboards)

Standardized dashboards are currently available for [Forms](https://va-gov.domo.com/page/447193050?userId=853813555), [Content](https://va-gov.domo.com/page/426422632), & [Search](https://va-gov.domo.com/page/1964748112) products only. To see if you should request a standardized dashboard, check [Should I request a standardized KPI dashboard](https://docs.google.com/presentation/d/16mRcslbkWtW44rlsfQGXZnHtQ3GoLjn6GpFnd3Gvva8/edit#slide=id.p). If you’re not sure which dashboard(s) your product(s) qualify for, check the [Eligibility Dashboard](https://va-gov.domo.com/page/1550814739).

If your VA.gov product is not eligible for a standardized dashboard, you may be able to request an “edge case” dashboard. Please reach out to the Platform Analytics & Insights team on #vfs-platform-support for more information on this process.

### Expected delivery timelines

The Analytics team will notify you once this ticket has been pulled into a sprint. 

### For New Dashboards

In order to pull the correct data for your product, we need your help to define what pages and events to include, in addition to basic product info. We’ll need a filled out [KPI request template](https://docs.google.com/spreadsheets/d/1gipo-h3GCeVPsw6t6tEHJM5dtJMvotVm0LUgbpAeqZY/edit#gid=981224988) before we can pull this ticket into our sprint.

- [ ] Make a copy of the [Google Sheet](https://docs.google.com/spreadsheets/d/1gipo-h3GCeVPsw6t6tEHJM5dtJMvotVm0LUgbpAeqZY/edit#gid=981224988)
- [ ] Complete worksheet as instructed
- [ ] Link your completed spreadsheet here: ______
- [ ] Check your sharing settings to make sure the Analytics team will be able to access it
- [x] **Add your team label to follow progress**

### For Updates to Existing Dashboards 

First, tell us what type of change you need to make:

- [ ] Change basic product info
- [ ] **Forms**: funnel changes
- [ ] **Content**: Add or remove pages included
- [x] **Other**: **_We'd like a filter added to the contact info chart - we only need this to start as of February 1st. We started passing the error key just before february first so we wouldn't have data until then. We need the "error key" added into our data set for https://va-gov.domo.com/page/1834995012/kpis/details/2068575095 - It should be custom dimension 57_**

Next, use the original [KPI request template](https://docs.google.com/spreadsheets/d/1gipo-h3GCeVPsw6t6tEHJM5dtJMvotVm0LUgbpAeqZY/edit#gid=981224988) you filled out to update your information. If you are not able to find that spreadsheet, reach out to the Analytics team on #vfs-platform-support for a new spreadsheet that reflects what we currently have on file.

**If you cannot locate your original worksheet you can create a new one:**
- [ ] Make a copy of the [Google Sheet](https://docs.google.com/spreadsheets/d/1gipo-h3GCeVPsw6t6tEHJM5dtJMvotVm0LUgbpAeqZY/edit#gid=981224988)
- [ ] Complete worksheet as instructed
- [ ] Link your completed spreadsheet here: ______
- [ ] Check your sharing settings to make sure the Analytics team will be able to access it
- [ ] **Add your team label to follow progress**

**Additional Info For Both Options**

- Please use the [KPI Framework](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/analytics/Analytics%20Playbook/va-gov-platform-analytics-kpi-framework.pdf) for more information on the metrics used in the Standard KPI dashboards. 
- If you need any analytics support, read through the [How to request Analytics support](https://depo-platform-documentation.scrollhelp.site/analytics-monitoring/Analytics-customer-support-guidelines.1586823275.html#Analyticscustomersupportguidelines-4.RequestaKPIDashboardinDomo).
- If you do not have Domo access, please [request access here](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=joanneesteban&labels=analytics-insights%2C+analytics-request%2C+access-request&template=analytics-request-google-analytics-domo-access.md&title=Request+access+to+Google+Analytics+and%2For+Domo).

---

### Acceptance Criteria

- [ ] Filter has been successfully added to the, **Contact Info Chart**.
- [ ] Filter starts as far back of at least February 1st.
- [ ] Confirm "error key" added into our data set for https://va-gov.domo.com/page/1834995012/kpis/details/2068575095 - It should be custom dimension 57
- [ ] Report edits have been QA'd successfully
- [ ] Update has been shared back to requestor

**Comment by ChristineYoungZA** [comment]
Hi @mtcA6 could you give more details of how to obtain the error key code, like what variable to reference the custom dimensions. The dataset that feeds this does not have anything referenced as custom dimensions.

**Comment by jestutt** [comment]
@ChristineYoungZA  I think this requires an update in the BigQuery pull to add the Error Key (GA UA Custom Dimension #57) before the report can be updated.

**Comment by mtcA6** [comment]
@jestutt @ChristineYoungZA Do you all need another ticket? or do you need me to join office hours tomorrow afternoon to chat through this?

**Comment by mtcA6** [comment]
screenshot we were talking about during office hours 
![image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/129431463/1bdcab88-e224-4562-90c2-988807febda8)

**Comment by dorseypa1** [comment]
The BigQuery view, SELECT * FROM `vsp-analytics-and-insights.vsp_sites.vw_profile_detail`,  underlying the Domo dataset related to this ticket has been modified to include an error_key attribute.

**Comment by ChristineYoungZA** [comment]
@mtcA6 I've created an extra chart in the dashboard under the section Error Key with the error keys broken out by the specific codes referenced in the screenshot.

**Comment by mtcA6** [comment]
Are these just the error keys fired as part of the contact information api (so i think user V0)? If so can we rename that section to more clearly relate it to the contact info errors in that prior chart?  

![image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/129431463/e91d5783-4fe1-458f-87ab-4a4c3260daa2)

**Comment by ChristineYoungZA** [comment]
Hello @mtcA6 I've updated the title to reflect just error keys related to contact information. We will go ahead and close this as completed.

**Comment by mtcA6** [comment]
This is great! Thanks for the help, closing this out
